README.md

```diff
@@ -58,6 +68,232 @@ dependencies {
 
 > Please keep in mind that the Jitpack repository should to be added to the project gradle file while the dependency should be added in the module gradle file. (More about build configuration files is available [here](https://developer.android.com/studio/build))
 
+
+## Demo projects
+We've worked with Product and Design to create simple demo applications that showcase our SDKs, that you'll be able to run locally as soon as you clone the repository.
+
+You will find our demo apps in the root of the repository, inside respective folders:
+- `app` - Java and Imperative UI. Covers default style and 3DS.
+- `example_app_frames` - Kotlin and Declarative UI. Covers: default and custom style, theming, 3DS and Google Pay.
+
+Once running `example_app_frames`, you will find the home screen with a number of design options. We have tried to make them pretty contrasting to give your UI/UX teammates an idea of what can be achieved. We've also tried to write the code in the simplest way to track and understand how each UI flavour was created. Just start from `HomeScreen.kt` and follow the button actions in code for some examples on how we achieve those UIs.
+
+
+## Get started
+If you got here, we'll either assume you've completed Integration or you're just curious. If none, then please complete [Integration](#Integration) first.
+
+### Step 1: Create a callback for the Payment flow
+```kotlin
+val paymentFlowHandler = object : PaymentFlowHandler {
+    override fun onSubmit() {
```

Comment:
@alex-ioja-yang-cko @ehab-al-cko on iOS we don't do this as we are adding the progress bar on the pay button ourselves. This is a good idea if merchants what to add their own loading screen and dismiss when they get back the token.

Reply:
That is a very different workflow from what iOS does, I would have to seriously question why we are not synchronised on this cross platform?

iOS handles full UI flow, from starting the tokenisation to its outcome.
Sounds like here, Android handles only part of the UI flow, from starting the tokenisation to end of details being inputed?

Either way there is not much to do one or the other, but being same product I would have expected same behaviour?

Reply:
Not related to the PR, please move to slack discussion.